README.md

```diff
@@ -351,82 +210,8 @@ class Application(
 
   See also [Customising error responses for an authenticated API]().
 
-Both the actions add the current user to the request, this is available as `request.user`.
-
-In order to instantiate your controller you'll need an instance of PanDomainAuthSettingsRefresher. This one cane be manually created during the wiring of your application (application loader).
-Here's an example of instantiation:
-
-```scala
-  val awsCredentialsProvider = ...
-
-  val panDomainSettings = new PanDomainAuthSettingsRefresher(
-    domain = "local.dev-gutools.co.uk",
-    system = "example",
-    actorSystem = actorSystem,
-    awsCredentialsProvider = awsCredentialsProvider // optional as there's a default value to DefaultAWSCredentialsProviderChain
-  )
-
-  val controller = new AdminController(controllerComponents, configuration, wsClient, panDomainSettings)
-```
-
-### Using pan domain auth with another framework
-
-Other scala frameworks exist as well as play. Full framework libraries are not provided for these yet as we predominantly use play at the guardian.
-To use pan domain auth with another framework you will need to provide an equivalent of the user auth checking in the play actions and provide an
-implementation of the OAuth integration. Have a look at how this is done in the play library and provide your own implementation for
-your framework and http client etc.
-
-More examples and framework clients may be added in the future as they become available.
-
-### Configuring access to the S3 bucket
-
-Access to the S3 bucket is controlled by passing an [AWSCredentialsProvider](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AWSCredentialsProvider.html)
-and [region](https://docs.aws.amazon.com/general/latest/gr/rande.html) to the `PanDomainAuthSettingsRefresher`.
-
-### The user object
```

Comment:
I didn't see much value in describing the fields of `User` and `AuthenticatedUser`. They are self-descriptive and covered in the example code.